bundle/run.sh

```diff
@@ -162,6 +162,165 @@ certificate-generator --cmd.generateIdentityCertificate=$COAP_GATEWAY_CLOUD_ID -
 echo "generating NGINX cert"
 certificate-generator --cmd.generateCertificate --outCert=$EXTERNAL_CERT_DIR_PATH/$DIAL_FILE_CERT_NAME --outKey=$EXTERNAL_CERT_DIR_PATH/$DIAL_FILE_CERT_KEY_NAME --cert.subject.cn="localhost" --cert.san.domain="localhost" --cert.san.ip="0.0.0.0" --cert.san.ip="127.0.0.1" $fqdnSAN --signerCert=$CA_POOL_CERT_PATH --signerKey=$CA_POOL_CERT_KEY_PATH
 
+# copy ceritficates to paths
```

Comment:
What has this to do with `coap-gateway: update pending commands via eventbus`? :)